database.sql

```diff
@@ -1,6 +1,6 @@
 -- ------------------------------------------
--- Friendica 3.5.1-dev (Asparagus)
--- DB_UPDATE_VERSION 1213
+-- Friendica 3.5.1-rc (Asparagus)
```

Comment:
I you later merge back, you have **rc** in `develop` which is not what you want.